### ISSUE TITLE
[Bug] Focus Follows Cursor ignoring dropdowns as part of window.

### ISSUE THREAD

**Issue by drazil100** [pop-os/cosmic-epoch#989]
Cosmic is my first experience with Focus Follows Cursor so I don't know how others do it, but when a dropdown extends past the border of a window and you move your mouse over the dropdown the focus shifts to the window under the dropdown making mouse selection impossible.

**Comment by nevdelap** [comment]
Yeah, I find I have to full screen the window, or scroll the content, or something, to get the menu to stay within that window.

**Comment by MarkHughes1** [comment]
I found the same with Firefox, I have some menus with lots of bookmarks and if the menu is long enough to overlap the taskbar then I can't click that menu item instead the icon on the taskbar below will be clicked.

**Comment by drazil100** [comment]
I would like to add I had set my delay MS to 0 and causes the dropdowns from the panel applets to close. When I moved it up to 100 ms I could safely traverse the chasm between the panel and the dropdown without the dropdown closing on me.
